catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/HadoopCatalogOperations.java

```diff
@@ -716,8 +733,9 @@ private Path getSchemaPath(String name, Map<String, String> properties) {
 
   @VisibleForTesting
   Path formalizePath(Path path, Map<String, String> configuration) throws IOException {
-    FileSystem defaultFs = getFileSystem(path, configuration);
-    return path.makeQualified(defaultFs.getUri(), defaultFs.getWorkingDirectory());
+    try (FileSystem defaultFs = getFileSystem(path, configuration)) {
+      return path.makeQualified(defaultFs.getUri(), defaultFs.getWorkingDirectory());
+    }
```

Comment:
Why wrap this into a try block?

Reply:
I just want to use the `try-with-resource` mechanism to close `FileSystem` automatically.